### DISCUSSION TITLE
How can disable the auto docs showing up when writing?

### DISCUSSION THREAD

**Discussion by banjo** [LazyVim/LazyVim#3700] [Q&A] [answered]
Currently, when writing TypeScript, I get up the docs for functions when writing them. I don't want that to happen automatically, I just want to trigger it using "K". 

[Here it is in action (too big GIF for github)](https://i.imgur.com/JANnnlN.gif)

This is the docs view I'm talking about:
![CleanShot 2024-06-17 at 13 22 07@2x](https://github.com/LazyVim/LazyVim/assets/21983557/2e490a25-05e7-4c26-9509-9712128e6243)

**Comment by dpetka2001** [comment, marked as answer]
Try this to see if it does what you want
```lua
-- ~/.config/nvim/lua/plugins/noice.lua
return {
{
  "folke/noice.nvim",
  opts = {
    lsp = {
      signature = {
        auto_open = {
          enabled = false,
        },
      },
    },
  },
},
}
```

**Reply by banjo** [reply]
That did the trick. Thank you!
